### PR TITLE
LPS-149016 [CP 2.0] Verify Analytics Cloud Activation setup status to skip on onboarding flow

### PR DESCRIPTION
**Pull request by igordmf**

### CODE REVIEW

**Comment by igordmf** [comment]
ci:test

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js [hunk 1]
```diff
@@ -31,20 +34,49 @@ const Pages = () => {
...
+			subscriptionAnalyticsCloud &&
+			!analyticsCloudActivationSubmittedStatus
+		) {
+			return dispatch({
+				payload: ONBOARDING_STEP_TYPES.analyticsCloud,
+				type: actionTypes.CHANGE_STEP,
+			});
 		}
+		window.location.href = PAGE_ROUTER_TYPES.project(project.accountKey);
+	};
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:63]: Should be good use React Router here, to centralize the logic to redirect pages :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js [hunk 2]
```diff
@@ -31,20 +34,49 @@ const Pages = () => {
...
+			return dispatch({
+				payload: ONBOARDING_STEP_TYPES.analyticsCloud,
+				type: actionTypes.CHANGE_STEP,
+			});
+		}
+		window.location.href = PAGE_ROUTER_TYPES.project(project.accountKey);
+	};
+
+	const analyticsCloudPageHandle = () => {
+		window.location.href = PAGE_ROUTER_TYPES.project(project.accountKey);
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:79]: Should be good use React Router here, to centralize the logic to redirect pages :)

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js (hunk 1 above)
Comment (igordmf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:63]: Hey man, is the approach that we already use on the onboarding flow, I just replied for the others steps

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js (hunk 2 above)
Comment (igordmf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:79]: Hey man, is the approach that we already use on the onboarding flow, I just replied for the others steps

**Comment by jairmedeiros** [comment]
ci:stop

**Comment by jairmedeiros** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+						Done
+					</Button>
+				),
+			}}
+			headerProps={{
+				helper: successTexts[productType].helper,
+				title: successTexts[productType].title,
+			}}
+		>
+			<div className="container font-weight-bold pl-6 pr-6 pt-9 text-center">
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js:50]: I believe you can put just px-6 here

**Review by claricebrasil — APPROVED** [review]
Good job!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js (hunk 3 above)
Comment (igordmf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js:50]: Hey Clarice, I agreed with you! It was my fault, I didn't pay attention on that.. It is a component that I just made it generic for sucess cases. The PR it`s on forwarding process, so I can adjust this after. Thank you!

**Pull request closed without merging**